cfme/containers/container.py

```diff
@@ -58,3 +58,13 @@ def get_detail(self, *ident):
         """
         self.load_details(refresh=True)
         return details_page.infoblock.text(*ident)
+
+    @staticmethod
+    def get_container_names():
+        sel.force_navigate('containers_containers')
+        return map(lambda r: r.name.text, list_tbl.rows())
+
+    @staticmethod
+    def get_pod_names():
+        sel.force_navigate('containers_containers')
+        return map(lambda r: r.pod_name.text, list_tbl.rows())
```

Comment:
It's under the Container object though and I can get either container names or pod names here. I need to differentiate somehow and that's why I opted for get_container_names and get_pod_names